safekeeper/src/timeline.rs

```diff
@@ -329,7 +329,7 @@ pub struct Timeline {
     mutex: RwLock<SharedState>,
     walsenders: Arc<WalSenders>,
     walreceivers: Arc<WalReceivers>,
-    timeline_dir: Utf8PathBuf,
+    pub timeline_dir: Utf8PathBuf,
```

Comment:
Please don't make it `pub`, there is `FullAccessTimeline::get_timeline_dir`.

Reply:
done